changelog/2024-06-18T11_16_48+02_00_add_time_types

```diff
@@ -0,0 +1 @@
+ADDED: Introduced a new module to more easily represent time and its relation to your clock domain.
```

Comment:
Please name the module you added.

Reply:
Or if you move them to an existing module, name the types.

Reply:
All types?

Reply:
You can also link to the relevant section of the Haddock. Or just say "`Seconds` and similar types", phrased a bit better. It's just that now you say "we added something to more easily represent time". But if I'm a Clash user and I'm reading the changelog, how am I going to find these new toys? There's no clue where I should look. If you mention `Seconds`, they'd probably find the rest.

Reply:
Ah now you say "added to `Clash.Signal.Internal`" but that's not really helpful either. First off, we shouldn't be directing users to plow trough internal modules. Secondly, that module is fairly large, so it's still a hunt.